BUILD.bazel

```diff
@@ -993,13 +1001,18 @@ ray_cc_library(
         ],
     }),
     deps = [
-        ":node_manager_fbs",
+        ":raylet_client_connection_lib",
         ":node_manager_rpc",
-        ":ray_common",
+        "//src/ray/common:id",
+        "//src/ray/common:asio",
+        "//src/ray/common:ray_object",
+        "//src/ray/common:status",
         "//src/ray/common:network",
+        "//src/ray/common:task_common",
+        "//src/ray/util:util",
+        "//src/ray/util:logging",
+        "//src/ray/util:process",
         "//src/ray/protobuf:gcs_cc_proto",
-        "//src/ray/util",
```

Comment:
Better to add another dependency (which is implicitly included as of now): `//src/ray/protobuf:common_cc_proto`

Reply:
Changed.